### PR TITLE
Gemnasium

### LINKED ISSUE #1020: Set up gemnasium
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**
Set up gemnasium to monitor dependency vulnerabilities

* **What feature or behavior is this required for?**
~ security ~

Tagging in @DarthHater - can you handle?

### PR DESCRIPTION
**Pull request by DarthHater**
I rule and have completed some work on Case Manager that's ready for review!

This just adds a few public notes about our use of Gemnasium, which was largely accomplished via the Gemnasium page.

This pull request makes the following changes:
* Small changes to README and DOCS

Fixes #1020

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -26,4 +30,6 @@ test:
     - gem install --no-rdoc brakeman; bundle exec brakeman --exit-on-warn .
     - gem install --no-rdoc ruby_audit; bundle exec ruby-audit check
     - gem install --no-rdoc bundler-audit; bundle-audit update; bundle-audit check
+    - gemnasium auth login --token $GEMNASIUM_TOKEN
```
Comment (colinxfleming) [inline circle.yml]: Can you make this a one liner with the autoupdate script? e.g. `gemnasium auth login && bash autoupdate.sh`?

File: scripts/autoupdate.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (colinxfleming) [inline scripts/autoupdate.sh]: So it's a little clearer, can we call this `gemnasium_autoupdate.sh`?

File: docs/SECURITY.md [hunk 3]
```diff
@@ -22,6 +22,9 @@ Our project uses a variety of industry standard methods to ensure our existing c
 
 2. Because difficult-to-read code can lead to security holes, we regularly revise our code to conform to a standard style. We also make it a point to revise code in pull requests during code review.
 
+## Dependency checking
```
Comment (colinxfleming) [inline docs/SECURITY.md]: 👍

File: circle.yml (hunk 1 above)
Comment (DarthHater) [inline circle.yml]: Yeah I want to, just testing out if the variable is available first.

File: scripts/autoupdate.sh (hunk 2 above)
Comment (DarthHater) [inline scripts/autoupdate.sh]: Sure thing!

**Comment by DarthHater** [comment]
There are some oddities with how ENV variables are accessed with CircleCI. Opened a ticket for the time being to see what all is going on: https://discuss.circleci.com/t/gemnasium-and-circleci-gemnasium-token-seems-to-not-get-picked-up/12294

**Comment by colinxfleming** [comment]
I'm good with this when we work thru the issue -- let me know if there's anything I can do to be helpful.

**Comment by DarthHater** [comment]
Closed in favor of #1038

**Pull request closed without merging**